enterprise/frontend/src/embedding-sdk/errors/generic.ts

```diff
@@ -10,13 +10,17 @@ export function USER_FETCH_FAILED() {
 export function CANNOT_CONNECT_TO_INSTANCE({
   instanceUrl,
   status,
+  message,
 }: {
   instanceUrl: string;
   status?: number;
+  message?: string;
 }) {
+  const defaultMessage = `Unable to connect to instance at ${instanceUrl}${status ? ` (status: ${status})` : ""}`;
+
   return new MetabaseError(
     "CANNOT_CONNECT_TO_INSTANCE",
-    `Unable to connect to instance at ${instanceUrl}${status ? ` (status: ${status})` : ""}`,
+    message ?? defaultMessage,
```

Comment:
If we already have a good error message from the server, there is no need to also show a generic "Unable to connect...".